lib/taurus/core/tango/tangovalidator.py

```diff
@@ -176,7 +176,20 @@ def nonStrictNamePattern(self):
         '''In non-strict mode, allow double-slash even if there is no Authority.
         (e.g., "tango://a/b/c" passes this non-strict form)
         '''
-        return self.namePattern.replace('tango):(', 'tango)://(')
+        pattern = r'^((?P<scheme>%(scheme)s)://)?' + \
+                  r'((?P<authority>%(authority)s)(?=/))?' + \
+                  r'(?P<path>%(path)s)' + \
+                  r'(\?(?P<query>%(query)s))?' + \
+                  r'(#%(fragment)s)?$'
+        authority = '(?P<host>([\w\-_]+\.)*[\w\-_]+):(?P<port>\d{1,5})'
+        path = '/?(?P<devname>((?P<_devalias>(?<=//)([^/?#:]+))|' + \
```

Comment:
The lookbehind for the double-slash here is reproducing a bug in taurus3, not a feature. It is true that the latest implementation in Taurus3 would not allow `tango://alias`, but that is a "bug" (since according to the taurus3 docs it should be correct). In these cases we decided that the backwards compatibility layer should follow (if possible) the doc rather than the implementation.